packages/advanced-logic/src/extensions/payment-network/chainlink-supported-currencies.ts

```diff
@@ -0,0 +1,49 @@
+import { RequestLogicTypes } from '@requestnetwork/types';
```

Comment:
Not 100% sure but why not naming the file `conversion-supported-currencies.ts` instead ? It does not have to be Chainlink dependent.